tasks/host_volume.yml

```diff
@@ -0,0 +1,8 @@
+- name: create volume {{ item['name'] }}
+  file:
+    path: "{{ item['path'] }}"
+    owner: "{{ nomad_user }}"
+    group: "{{ nomad_user }}"
```

Comment:
We shouldn't assume group corresponds to member. EIther:
`item['group'] || nomad_group` (see above) or if nothing else `nomad_group`